_includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/make-src-file.adoc

```diff
@@ -0,0 +1,5 @@
+Now that you have a series of statements that's doing the right thing, the last step is to put them into a file so that they can be used outside the CLI session. Create a file at `src/statements.sql` with the following content:
```

Comment:
"that's doing the right thing" is vague.  Can this be more descriptive?

Reply:
ack